### ISSUE TITLE
training error rate or test error rate?

### ISSUE THREAD

**Issue by Icespellcaster** [davda54/sam#1]
Hello, the error rate of cifar100 in your paper is 3.92%. Is this the training error rate, or is it the error rate obtained by applying the trained model to the test set? Looking forward to your reply.

**Comment by davda54** [comment]
Hi, I'm not one of the authors, this is an unofficial implementation :) But I believe they report the test error rate because they also claim it is the SoTA performance. Besides, it should also be fairly easy to get to 0.00% on training set when finetuning a large pretrained model. I agree that they should make it more explicit, this is quite confusing.

**Comment by PForet** [comment]
Hi! Author here. The 3.92% figure is indeed the test error rate, obtained by finetuning an Efficientnet-l2 on cifar100.
